packages/docs/docs/guides/brand-new.md

```diff
@@ -4,7 +4,7 @@ sidebar_position: 1
 
 # Guide for AI Newcomers
 
-Large Language Models (LLMs) are powerful tools, representing a paradigm shift in how we build and use software. Machines now have the ability to reason and understand natural language and code. We predict over the coming years, incumbents will be either remake themselves, or be disrupted by AI-native products and platforms.
+Large Language Models (LLMs) are powerful tools, representing a paradigm shift in how we build and use software. Machines now have the ability to reason, understand natural language, and code. We predict over the coming years, incumbents will be either remake themselves, or be disrupted by AI-native products and platforms.
```

Comment:
", and code" feels weird

Reply:
", and to code" maybe then? Or just revert it back?

Reply:
`and write code`

Reply:
"... and understand natural language as well as source code."